lib/rules/no-literal-string.js

```diff
@@ -62,7 +62,7 @@ module.exports = {
       options: [option]
     } = context;
     const whitelists = [
-      /^[^A-Za-z]+$/, // ignore not-word string
+      /^[0-9!-/:-@[-`{-~]+$/, // ignore not-word string
```

Comment:
It should contain number and not-letter ascii characters. @megos

Reply:
Need to include whitespace?  
I think it's okay because test passed...  
<https://github.com/edvardchen/eslint-plugin-i18next/blob/99ed9f2804362737b9dbd12d8f01414a6b61b59a/tests/lib/rules/no-literal-string.js#L57>

Reply:
That doesn't seem necessary. I just don‘t know `-` can be used in this way than `0-9` or `a-z`.I merge it